cscs-checks/microbenchmarks/gpu/shmem/shmem.py

```diff
@@ -48,3 +60,63 @@ def __init__(self):
     @sn.sanity_function
     def num_tasks_assigned(self):
         return self.job.num_tasks
+
+    @sn.sanity_function
+    def assert_num_gpus(self):
```

Comment:
We should be consistent with these names. In other places we call it `assert_count_gpus()`.